administrative-sdk/pronunciationAnalysis.js

```diff
@@ -230,246 +215,169 @@ class PronunciationAnalysis {
    *
    * @param {its.PronunciationChallenge} challenge The pronunciation challenge to perform.
    * @param {its.AudioRecorder} recorder The audio recorder to extract audio from.
-   * @param {Sdk~pronunciationAnalysisPreparedCallback} [preparedCb] The callback that signals server is prepared for receiving data.
+   * @param {Sdk~pronunciationAnalysisPreparedCallback} [preparedCb] The callback that signals server is prepared for
+   *   receiving data.
    * @param {Sdk~pronunciationAnalysisCreatedCallback} [cb] The callback that handles the response.
    * @param {Sdk~pronunciationAnalysisCreatedErrorCallback} [ecb] The callback that handles the error response.
-   * @param {Sdk~pronunciationAnalysisProgressCallback} [progressCb] The callback that handles the intermediate results.
+   * @param {Sdk~pronunciationAnalysisProgressCallback} [progressCb] The callback that handles the intermediate
+   *   results.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.